components/datetime/src/scaffold/dynamic_impls.rs

```diff
@@ -38,7 +38,7 @@ impl DateInputMarkers for DateFieldSet {
     type DayOfMonthInput = datetime_marker_helper!(@input/day_of_month, yes);
     type DayOfYearInput = datetime_marker_helper!(@input/day_of_year, yes);
     type RataDieInput = datetime_marker_helper!(@input/rata_die, yes);
-    type DayOfWeekInput = datetime_marker_helper!(@input/day_of_week, yes);
+    type WeekdayInput = datetime_marker_helper!(@input/weekday, yes);
```

Comment:
Depending on how we feel about our policy, this might not be semver compatible because clients can stably refer to an associated type of a scaffold trait. It requires the word "scaffold" but not "unstable" in source